README.md

```diff
@@ -55,8 +55,63 @@ Our Developer Success Team can assist with answering any questions as well as pr
 
 [Forums, Support and Knowledge Base](https://community.playfab.com/hc/en-us)
 
+6. Example Usage (Template):
+----
+```
+function VanillaPFTemplate(bar)
+{
+    function PFTemplate(bar)
+    {
+        console.log("@ PFTemplate..");
+
+        var req =
+        {
+            // TODO: Replace "fu" with your own request JSON KVP's
+            "fu": bar
+        };
+
+        // TODO: Uncomment below and replace FuBar with 
+        // a proper PlayFabClientSDK.function(request, callback)
+
+        // Callback =>
+        function onPFTemplate(response, err)
+        {
+            console.log("@ onPFTemplate()..");
+            if ( response && response["code"] == 200 )
+            {
+                // SUCCESS >>
+                console.log( JSON.stringify(response) );
+                var data = response["data"];
+
+                if (data)
+                {
+                    var data = response["data"];
+                    // TODO: Handle success
+                }
+            }
+            else
+            {
+                // FAIL >>
+                console.log( JSON.stringify(err) );
+                var errMainCode = err["code"];
+                var errStatus = err["status"];
+                var errCode = err["errorCode"];
+                var errMsg = err["errorMessage"];
+
+                var errMainTxt = "**ERR " + errMainCode + " (" + errStatus + ") @ onPFTemplate: ";
+                var errPlayFabTxt = errMsg + "(" + errCode + ")";
+                console.log(errMainTxt + errPlayFabTxt);
```

Comment:
You can modify it to just use alert of course

On Sat, Aug 6, 2016, 1:39 AM Paul Gilmore <notifications@github.com> wrote:

> In README.md
> <https://github.com/PlayFab/JavaScriptSDK/pull/67#discussion_r73731079>:
>
> > +                    var data = response["data"];
> > +                    // TODO: Handle success
> > +                }
> > +            }
> > +            else
> > +            {
> > +                // FAIL >>
> > +                console.log( JSON.stringify(err) );
> > +                var errMainCode = err["code"];
> > +                var errStatus = err["status"];
> > +                var errCode = err["errorCode"];
> > +                var errMsg = err["errorMessage"];
> > +
> > +                var errMainTxt = "**ERR " + errMainCode + " (" + errStatus + ") @ onPFTemplate: ";
> > +                var errPlayFabTxt = errMsg + "(" + errCode + ")";
> > +                console.log(errMainTxt + errPlayFabTxt);
>
> I would prefer if the error handling were expanded a bit:
> https://github.com/PlayFab/JavaScriptSDK/blob/master/PlayFabApiTest.js#L131
>
> The code and status are useful too, but the "errorDetails" section (when
> present) can display much more useful information than the
> code/status/errorMessage alone.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PlayFab/JavaScriptSDK/pull/67/files/ad1284d3e056a64a056cc4bad6f3321ea9272267#r73731079>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIbjWBF1Jggwn47yYT62_3xSB9IkJdCfks5qc3VXgaJpZM4JRdIh>
> .
>